CallsheetLocalizations/de.xcloc/Localized Contents/de.xliff

```diff
@@ -555,6 +556,7 @@
       </trans-unit>
       <trans-unit id="Force Callsheet's theme, or use the system one." xml:space="preserve">
         <source>Force Callsheet's theme, or use the system one.</source>
+        <target state="translated">Erzwinge hellen oder dunklen Modus für Callsheet oder nutze die Systemeinstellung.</target>
```

Comment:
I came up with this translation, which seems a little less "threatening"

"Bestimme, wie Callsheet dargestellt wird, oder lass das System entscheiden."

Slack schreibt an der Stelle: 

"Wähle aus, ob Slack hell oder dunkel aussehen soll, oder übernimm die Einstellungen deines Geräts."

Vielleicht ein bisschen lang.

Reply:
Yeah, I think you're right!